docs/tech-notes/life_of_a_query.md

```diff
@@ -380,7 +377,7 @@ which initiates the processing, and
 [`Next`](https://github.com/cockroachdb/cockroach/blob/a83c960a0547720a3179e05eb54ea5b67d107d10/pkg/sql/plan.go#L149),
 which is called repeatedly to produce the next row.
 
-To tie this to the [SQL Executor](#SQL Executor) section above,
+To tie this to the [SQL Executor](https://github.com/cockroachdb/cockroach/blob/master/docs/tech-notes/life_of_a_query.md#sql-executor) section above,
```

Comment:
Can you make this just #sql-executor so that the link won't break if the document is moved?